volatility3/framework/plugins/windows/writecrashdump.py

```diff
@@ -0,0 +1,134 @@
+from typing import List, Type, Iterator, Tuple
+
+from volatility3.framework import interfaces, renderers, constants, exceptions
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import configuration, plugins
+from volatility3.framework.layers import physical, intel
+from volatility3.framework.symbols import intermed
+from volatility3.plugins.windows import info
+
+class WriteCrashDump(plugins.PluginInterface):
+    """Runs the automagics and writes the output to a crashdump format file"""
+
+    _required_framework_version = (2, 0, 0)
+    _version = (2, 0, 0)
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        return [
+            requirements.TranslationLayerRequirement(name = 'primary', description = 'Memory layer for the kernel',
```

Comment:
Do these two requirements need to be separate, or would making them a `ModuleRequirement` make more sense?  I'm kind of ok with either, but if there's no explicit reason for it, I'd prefer we keep things in sync and use a `ModuleRequirement`.

Reply:
Replaced with ModuleRequirement